packages/commonwealth/client/scripts/views/components/react_quill_editor/QuillLinkModal.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { CWTextInput } from '../component_kit/cw_text_input';
+import { CWButton } from '../component_kit/new_designs/CWButton';
+import {
+  CWModal,
+  CWModalBody,
+  CWModalFooter,
+  CWModalHeader,
+} from '../component_kit/new_designs/CWModal';
+
+type LinkModalProps = {
+  linkText: string;
+  linkUrl: string;
+  setIsModalOpen: (isOpen: boolean) => void;
+  setLinkText: (text: string) => void;
+  setLinkUrl: (url: string) => void;
+  handleAddLink: () => void;
+  isModalOpen: boolean;
+};
+
+export const AddLinkModal = ({
```

Comment:
We have a rule that name of the file, component, props and scss are all the same.

So LinkModal.tsx, LinkModal.scss const LinkModal  = () => {}, LinkModalProps etc

Please follow that rule in upcoming PRs

Reply:
i change the AddLinkModal to LinkModal and file name to LinkModal.tsx